app/src/keymap.c

```diff
@@ -115,8 +127,23 @@ int zmk_keymap_layer_toggle(uint8_t layer) {
     return zmk_keymap_layer_activate(layer);
 };
 
-bool is_active_layer(uint8_t layer, zmk_keymap_layers_state layer_state) {
-    return (layer_state & BIT(layer)) == BIT(layer) || layer == _zmk_keymap_layer_default;
+int zmk_keymap_layer_to(uint8_t layer) {
+    // Default layer is *always* active ; if going TO the default layer, need to short circuit this
+    // condition to ensure proper operation
+    if (layer != zmk_keymap_layer_default() && zmk_keymap_layer_active(layer)) {
```

Comment:
There's a bug in here; if layers 0, 1, 2, 3 are active, and going TO layer 1, layers 2 and 3 should be disabled.